packages/dds/tree/src/test/feature-libraries/optional-field/optionalChangeRebaser.test.ts

```diff
@@ -85,11 +85,11 @@ const OptionalChange = {
 			detach: ChangesetLocalId;
 		},
 	) {
-		return optionalFieldEditor.set(wasEmpty, ids);
+		return optionalFieldEditor.set(wasEmpty, ids, tag1);
 	},
 
 	clear(wasEmpty: boolean, id: ChangesetLocalId) {
-		return optionalFieldEditor.clear(wasEmpty, id);
+		return optionalFieldEditor.clear(wasEmpty, id, tag1);
```

Comment:
Instead of hard-coding the same tag for all the changes, you'll need to take the revision as a parameter.